src/sql/plot.py

```diff
@@ -244,9 +244,10 @@ def boxplot(payload, table, column, *, orient="v", with_=None, conn=None):
     if not conn:
         conn = sql.connection.Connection.current.session
 
-    payload[
+    if sql.connection.Connection.current:
+        payload[
             "connection_info"
-        ] = sql.connection.Connection._get_curr_connection_info()
+        ] = sql.connection.Connection.current._get_curr_connection_info()
```

Comment:
looks like a merge conflict, the new version got rid of this conditional, check out the master branch